abstract/group.go

```diff
@@ -51,6 +51,9 @@ type Secret interface {
 
 	// Set to a fresh random or pseudo-random secret
 	Pick(rand cipher.Stream) Secret
+
+	// SetBytes will take bytes and create a secret out of it
+	SetBytes([]byte) Secret
```

Comment:
Or when we rename abstract.Secret to abstract.Scalar, maybe nist.Int should become abstract.IntScalar, representing a "standard/default implementation" of the abstract.Scalar interface?